### ISSUE TITLE
loss is a very small value

### ISSUE THREAD

**Issue by yupeijei1997** [princeton-nlp/SimCSE#40]
hi，thank you for your excellent work.

I found, loss is a very small number from the beginning of training.

When my batch_size is 4, loss is about 0.1 of step 1.

I also tried the dropout enhancement method (batch_size=96) on my own data set. I also found that the loss was about 1.8 at the beginning, and it dropped to about 0.03 with only two steps. Due to the use of dropout, there are actually two doc/sentence's hidden is actually very close, so the loss is very small and easy to understand, but I am puzzled that such a small loss means that the gradient is also very small, which always feels a bit strange.

In other words, it feels that the pre-training model BERT can distinguish positive and negative cases very well from the beginning, so the loss is very small. So why this method can bring so much effect improvement.

I want to know how the loss size changes during your training. I want to confirm whether this situation I am talking about is a problem of model collapse.

**Comment by gaotianyu1350** [comment]
Hi,

In the unsupervised case, loss dropping is indeed rapid at the beginning of the training, and the model almost converges after a few hundred steps. As mentioned in our paper, the unsupervised training mainly helps with "uniformity", so it is normal that the loss is not very large. 

Also, to verify that the training goes well, you can check the validation results during training.

**Comment by yupeijei1997** [comment]
Hi，

Thanks for your answer. I'm sorry that I didn't reply in time due to the experiment being done this week, but after experimenting, I tried to use different temperature hyperparameters, and all got the same results as you said, and I trained normally on my own data set.
